agent/util-scripts/get-internal-tool

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+import sys
+import json
+from pathlib import Path
+
+if len(sys.argv) != 4:
+    print("Usage: get-internal-tool <metadata_location> <tool_name> <tool_type>")
+    sys.exit()
+
+meta_loc = Path(sys.argv[1])
+name = sys.argv[2]
+tool_type = sys.argv[3]
+
+try:
+    meta = json.load(open(meta_loc / "meta.json"))
+except Exception as exc:
+    print(f"Failed to load tool metadata: {exc}")
+    sys.exit()
```

Comment:
Might be cleaner to exit with 1 for failures, even if the caller doesn't care now..

And, yeah, more complicated; but a trampoline that calls a method and does a single `sys.exit` with the return status would make it flow better.